zerver/data_import/slack.py

```diff
@@ -80,13 +83,15 @@ def slack_workspace_to_realm(domain_name: str, realm_id: int, user_list: List[Ze
     realm['zerver_defaultstream'] = channels_to_zerver_stream_fields[0]
     realm['zerver_stream'] = channels_to_zerver_stream_fields[1]
     realm['zerver_huddle'] = channels_to_zerver_stream_fields[2]
-    realm['zerver_subscription'] = channels_to_zerver_stream_fields[5]
-    realm['zerver_recipient'] = channels_to_zerver_stream_fields[6]
+    realm['zerver_subscription'] = channels_to_zerver_stream_fields[6]
+    realm['zerver_recipient'] = channels_to_zerver_stream_fields[7]
     added_channels = channels_to_zerver_stream_fields[3]
     added_mpims = channels_to_zerver_stream_fields[4]
-    added_recipient = channels_to_zerver_stream_fields[7]
+    dm_members = channels_to_zerver_stream_fields[5]
+    added_recipient = channels_to_zerver_stream_fields[8]
```

Comment:
This block is kinda a mess.  It might be more readable if we just did 

(`realm['zerver_defaultstream], ...) = channels_to_zerver_stream_fields`

Or likely better, if we just passed the `realm` value into `channels_to_zerver_stream_fields` and had it do the setting.

(This can be a followup)